### DISCUSSION TITLE
Defect modules of pymatgen

### DISCUSSION THREAD

**Discussion by Jeff-oakley** [materialsproject/pymatgen#2936] [Q&A] [answered]
Do we still have defect modules in `pymatgen`? I cannot find the source code any more. It was merged from [`pyCDT`](https://github.com/mbkumar/pycdt) a few years ago. Due to some reason it disappeared from `pymatgen`.

**Comment by shyuep** [comment, marked as answer]
It is now in pymatgen-analysis-defects. That is separately maintained. You will have to contact those maintainers on any issues, missing functions or otherwise.
